frontend/src/components/BookCard.tsx

```diff
@@ -0,0 +1,26 @@
+import {Book} from "../models/Book";
+
+type Props = {
```

Comment:
Maybe you could name the type BookCardProps, so it would be absolutely clear, to what component the props belong. But the way you named ist, is also good way. :)